recipes/cineform/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+        "enable_tools": True
+    }
+    implements = ["auto_shared_fpic"]
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.settings.os == "Linux" and self.options.shared:
+            self.requires("libuuid/1.0.3", transitive_libs=True)
+
+    def validate(self):
+        if self.settings.arch not in ["x86_64"]:
+            raise ConanInvalidConfiguration(
+                "{}/{} supported only on x86_64 arch".format(self.name, self.version))
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.5.1 <4]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        apply_conandata_patches(self)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["BUILD_TOOLS"] = self.options.enable_tools
+        tc.cache_variables["BUILD_STATIC"] = not self.options.shared
+        tc.cache_variables["BUILD_SEPARATED"] = self.options.enable_separated
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE.txt", self.source_folder,
+             os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE-APACHE", self.source_folder,
+             os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE-MIT", self.source_folder,
+             os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.configure(variables={
+            "CMAKE_INSTALL_PREFIX": self.package_folder
+        })
+        cmake.install()
+        postfix = ".exe" if self.settings.os == "Windows" else ""
+
+        if self.options.enable_tools:
+            copy(self, "*TestCFHD"+postfix, src=self.build_folder,
+                 dst=os.path.join(self.package_folder, "bin"), keep_path=False)
+            copy(self, "*WaveletDemo"+postfix, src=self.build_folder,
+                 dst=os.path.join(self.package_folder, "bin"), keep_path=False)
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+        if self.options.shared and self.settings.os == "Windows":
+            mkdir(self, os.path.join(self.package_folder, "bin"))
+            if self.options.enable_separated:
+                shutil.move(os.path.join(self.package_folder, "lib", "CFHDDecoder.dll"), os.path.join(
+                    self.package_folder, "bin", "CFHDDecoder.dll"))
+                shutil.move(os.path.join(self.package_folder, "lib", "CFHDEncoder.dll"), os.path.join(
+                    self.package_folder, "bin", "CFHDEncoder.dll"))
+            else:
+                shutil.move(os.path.join(self.package_folder, "lib", "CFHDCodec.dll"), os.path.join(
+                    self.package_folder, "bin", "CFHDCodec.dll"))
```

Comment:
does `cmake.install()` not place the right files in the right place?

in general, try to respect the layout set by the upstream library, we're not here to "correct" it - just make sure the `package_info()` method reflects the actual package folder layout